eventsourcing/src/main/java/org/axonframework/eventsourcing/eventstore/jdbc/statements/AppendSnapshotStatementBuilder.java

```diff
@@ -47,5 +47,5 @@
      * @throws SQLException when an exception occurs while creating the prepared statement.
```

Comment:
I expect a `@param timestampWriter` to be added to the javadoc.

Reply:
Added.